frontend/app/chat/[chatId]/__tests__/page.test.tsx

```diff
@@ -72,6 +72,13 @@ vi.mock("@tanstack/react-query", async () => {
   };
 });
 
+vi.mock(
+  "../components/ActionsBar/components/ChatInput/components/ChatEditor/ChatEditor",
+  () => ({
+    ChatEditor: () => <div data-testid="chat-input" />,
```

Comment:
Can we try using 'userEvent' and actually texting the behaviour of the component for next time:
AAU, I am able to click on @ and the menu appears, I can manipulate the menu with the keyboard and click, after that it appears on the screen and I can write a message

Wdyt ?